### ISSUE TITLE
Definition of session_Id and user_Id

### ISSUE THREAD

**Issue by thoo1** [microsoft/ApplicationInsights-JS#1589]
**Is your feature request related to a problem? Please describe.**
We currently use session_Id and user_Id in application insights to correlate events for a user's session. We are having a problem with what session_Id and user_Id represents in terms of different tabs (sessions) and different browser instances and also across different browsers.

Our current understanding:
session_Id: unique across browser tabs as each browser tab is a new session
user_Id: unique across browsers (not across different browser windows) - for example different id in firefox and edge

What we are observing:
session_Id: is not unique across browser tabs. Each browser tab actually shares the same session_Id. Session_Id also has some default expiration so even if user is still using the same browser tab, there may be multiple session_Ids
user_Id: consistent with our understanding, it is unique across different browsers

And of course if user chooses to clear cookies and/or storage, we expect the ids to not persist

**Describe the solution you'd like**
I couldn't find documentation on the readme for what session_Id and user_Id represents but I did find some configuration for session_Id expiration and renewal.

I would like a clearer explanation on what session_Id and user_Id represent and are designed for so we atleast can align our understanding. As a stretch (and probably breaking change), we would want to use session_Id like how we would like (unique across browser tabs, does not change within each browser tab)

**Describe alternatives you've considered**
Have not considered much yet, I will look into seeing if we can do this ourselves by changing some config

**Additional context**
It can get confusing to debug logs if the session_Id changes mid session, we are not sure what it means.

**Comment by MSNev** [comment]
Yes, both are persistent cookies (because of the expiry time), so they will be shared across all instances of the same browser up until the cookies are cleared or expire.

There is a "namePrefix" config that different applications can use to keep the session separate, and there may some Azure documentation around somewhere (not explicitly linked to the JS SDK -- maybe just the portal). But I'll tag this as a documentation update as there is no code change required -- just as you point out some clarification around these cookies

There may be some existing internal documentation I can ping you over teams
